test/jdk/tools/jar/MultipleManifestTest.java

```diff
@@ -199,6 +199,10 @@ private void jar(String cmdline) throws IOException {
         }
     }
 
+    private void assertOutputContains(String expected) {
+        Assertions.assertTrue(baos.toString().contains(expected));
```

Comment:
The linked JBS issue notes that the test was failing because of the additional (new) log messages from the jar tool itself. So I think the logs will be deterministic (since they aren't being generated by something else other than the jart tool). Using `contains()` thus looks fine to me.

Reply:
Do you think we should print out the actual and expected, when they don't match? It might make it easier to debug any unexpected failures in future.

Reply:
The output is always printed in the log, so we know the actual value. If a test failed, the source code tell us the expected.

Reply:
Thank you Henry, I believe that's good enough.